### DISCUSSION TITLE
my inbox can't be cleard

### DISCUSSION THREAD

**Discussion by HomyeeKing** [community/community#120940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body
the inbox keeps unread status for months, but I've already read all of them

![image](https://github.com/community/community/assets/49113249/69950dd2-4f7a-41fd-979c-3460a725c8df)

this is my inbox 
![image](https://github.com/community/community/assets/49113249/59157eef-c006-4e15-ba3c-ca4fd01dc756)

**Comment by Manjunathhs-0003** [comment]
yeah after you read them, if you hover over the last part of the row
you get this tick mark kinda thing
just check that thing and it will be gone

**Comment by Manjunathhs-0003** [comment]
<img width="1220" alt="Screenshot 2024-04-24 at 13 13 23" src="https://github.com/community/community/assets/146455138/8c52d06b-8fdc-4fe1-9196-a9b290fc5598">
just like this, the tick mark

**Reply by HomyeeKing** [reply]
em.. I have no message to tick

**Comment by Manjunathhs-0003** [comment]
then it might be glitch, just logout and log in back

**Reply by HomyeeKing** [reply]
still there :/

**Reply by Manjunathhs-0003** [reply]
:( sad
